sections/04_study.md

```diff
@@ -188,139 +188,91 @@ disease, which will lead to novel diagnostics and therapeutics.
-where they could easily mutate, add, or delete base pairs in a sequence and see
-how the model changed its prediction. This is something that would be very time
-consuming in a lab setting, but easy to simulate using their model.
-Visualization techniques on deep learning models are important because they can
-provide new insights on regulatory mechanisms and can lead biologists to test
-and verify in a lab setting, leading to new biomedical knowledge. Since the
-"linguistics" of DNA are unclear, interpretability of models is crucial to
-pushing our understanding forward.
-
-`TODO: cut RNA-binding proteins from above section, refer to representative
-papers`
+large number of genomic sequences. Lanchantin et al. [@tag:Lanchantin2016_motif]
+introduced several new convolutional and recurrent neural network models that
+further improved TFBS predictive accuracy. Due to the motif-driven nature of the
+TFBS task, most architectures have been convolutional-based
+[@tag:Zeng2016_convolutional]. While many models for TFBS prediction resemble
+computer vision and natural language processing (NLP) tasks, it is important to
+note that DNA sequence tasks are fundamentally different than NLP tasks, and
+thus the models should be adapted from traditional deep learning models in order
+to account for such differences. For example, motifs may appear in either strand
+of a DNA sequence, resulting in two different forms of the motif (forward and
+reverse complement) due to complementary base pairing. To handle this issue,
+Shrikumar et al. [@tag:Shrikumar2017_reversecomplement] created a convolutional
+model that can find motifs in both directions. Since deep learning for protein
+binding prediction is still in early stages, we expect to see an increase in
+domain-specific architectures for this task.
+
+Despite these advances, several challenges remain. First, because the input data
+(ChIP-seq measurements) are continous and the output (whether or not there is TF
+binding at a particular site) is binary, false positives or false negatives can
+result depending on the threshold chosen by the algorithm. Second, most datasets
+for TFBS prediction only assay one TF at a time, but in reality multiple TFs may
+bind at the same location in a cooperative or antagonistic manner. Deep learning
+can be useful in integrating different datasets and output one prediction
+[@tag:Zhou2015_deep_sea]. Third, it is unclear exactly how to define a
+non-binding or "negative" sites in the training data. Since the number of
+positive binding sites of a particular TF is relatively small with respect to
+the total number of base-pairs in DNA, we must choose a small subset of the
+total non-binding sites, resulting in some sort of bias over all of the actual
+negative sites. Regardless of the negative site selection, most datasets evenly
+balance the positive and negative binding sites and report auROC for the metric.
```

Comment:
We've used AUC ROC elsewhere instead of auROC.  I'll note that in #345 and we can make ti consistent later.